shell/platform/darwin/macos/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -608,6 +618,86 @@ - (bool)testAutocompleteDisabledWhenAutofillGroupIncludesPassword {
   return true;
 }
 
+- (bool)testContentTypeWhenAutofillTypeIsUsername {
+  // Set up FlutterTextInputPlugin.
+  id engineMock = flutter::testing::CreateMockFlutterEngine(@"");
+  id binaryMessengerMock = OCMProtocolMock(@protocol(FlutterBinaryMessenger));
+  OCMStub(  // NOLINT(google-objc-avoid-throwing-exception)
+      [engineMock binaryMessenger])
+      .andReturn(binaryMessengerMock);
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:engineMock
+                                                                                nibName:@""
+                                                                                 bundle:nil];
+  FlutterTextInputPlugin* plugin =
+      [[FlutterTextInputPlugin alloc] initWithViewController:viewController];
+
+  // Set input client 1.
+  [plugin handleMethodCall:[FlutterMethodCall
+                               methodCallWithMethodName:@"TextInput.setClient"
+                                              arguments:@[
+                                                @(1), @{
```

Comment:
Consider moving these dictionaries into a variable for readability. This is a little hard to read due to how tabbed over this is.

Reply:
Will leave it in this patch, for consistency with the surrounding code, but will follow up with a refactoring patch where I extract this to a factory function, since we do this all through the test file.